src/pages/detail.css

```diff
@@ -0,0 +1,88 @@
+
+.detail-section {
+    position: relative;
+    margin: 0;
+    padding: 0;
+}
+
+.detail-section .back-button {
+    position: absolute;
+}
+
+.back-button {
```

Comment:
Your back button seems to be very far out to the side. Personally I think it would look better if you gave the button some margin to move it both down and to the right, or maybe adding padding to the entire body?